core/src/main/resources/jenkins/tools/GlobalToolConfiguration/index_ru.properties

```diff
@@ -0,0 +1,5 @@
+Back\ to\ Dashboard=Вернуться к главной странице
```

Comment:
there are actually a bunch of disordered translations for "dashboard". "Вернуться к главной странице" which i found and duplicated, "На главную", "Домой". "Dashboard" itself is "ИнфоПанель" in one place. tbh, it's kind of a mess and once again we should probably stick to one and unify

Reply:
I was leaning towards `панель управления` but now that I think of it it seems different (`Control panel` in Windows for example). Guess that makes `панель задач` a better option.
Maybe it can be left out to not block this PR and unified as a separate PR.

Reply:
"панель задач" also makes me kind of think of windows taskbar, so I guess it's on the same level as "панель управления", with a potential alternate meaning.